.github/workflows/tests-e2e-upgrade.yaml

```diff
@@ -432,6 +432,20 @@ jobs:
             ciliumendpointslice: 'true'
             skip-upgrade: 'true'
 
+          - name: '23'
```

Comment:
This should be `27` and `max-parallel: 26` should also be changed to `max-parallel: 27`

Reply:
Good catch, thanks! Should be good now.